index.js

```diff
@@ -2,25 +2,30 @@
 const { ProvidePlugin } = require("webpack")
 const filterObject = require("filter-obj")
 
-const excludeObjectKeys = (object, excludeKeys) => filterObject(object, key => !excludeKeys.includes(key))
+const filterObjectKeys = (object, keys, isInclude) => filterObject(object, key => keys.includes(key) === isInclude)
 
 module.exports = class NodePolyfillPlugin {
-	constructor(options = {}) {
-		this.options = {
-			excludeAliases: [],
-			...options
+	constructor({ excludeAliases = [], includeAliases = [] } = {}) {
```

Comment:
There shouldn't be destructuring here because then we'd have to reference all future options twice: Once here and a second time when setting `this.options`.